modules/exploits/linux/http/panos_auth_rce.rb

```diff
@@ -0,0 +1,140 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Palo Alto Networks Authenticated Remote Code Execution',
+        'Description' => %q{
+          An OS Command Injection vulnerability in the PAN-OS management interface that allows authenticated administrators to execute arbitrary OS commands with root privileges.
+
+          This issue impacts:
+
+          PAN-OS 9.0 versions earlier than 9.0.10;
+
+          PAN-OS 9.1 versions earlier than 9.1.4;
+
+          PAN-OS 10.0 versions earlier than 10.0.1.
```

Comment:
Have you run this against a linter? The formatting here is very odd and you have a mix of long lines and lines that could likely be merged into one line. I'd look at reformatting this to be easier to read in smaller consoles.